src/pages/index.tsx

```diff
@@ -25,6 +25,7 @@ const HomePage = ({ data, error }: HomePageProps) => {
 
   return (
     <div>
+      <EventCard data={data.eventsSection} />
```

Comment:
this won't be used here, but inside the events component (the one which displays two/or one on mobile) - you can leave it there for now if you want to work on the events component overall too 👍

Reply:
Yes, I'm gonna work on events component.